src/locales/en/messages.json

```diff
@@ -3040,5 +3040,17 @@
   },
   "userPreference": {
     "message": "User Preference"
+  },
+  "vaultTimeoutAction": {
```

Comment:
@kspearrin @cscharf @mportune-bw Any verbiage suggestions? I followed existing patterns but I'm always open for changes that everyone agrees on.

Reply:
The only thing I would say is that the other properties, variables, methods, etc. exclude the term "vault" as they already apply to the vault in principal. Since this replaces 1st class citizen concepts like "lock" which were only "lock" or "lockOptions", etc. perhaps "timeoutAction" would be enough, vs. the more verbose/explicit naming convention implemented here. Just my $0.03 (adjusted for inflation).

Reply:
I agree we're a little verbose on the naming here, but that's outside the scope of this PR. We can consider renaming things "timeoutService", "timeout", "timeoutAction", etc in a future PR if it makes sense to everyone.